frontend/projects/upgrade/src/app/core/http-interceptors/http-error.interceptor.spec.ts

```diff
@@ -77,25 +77,6 @@ describe('HttpErrorInterceptor', () => {
   });
 
   describe('#intercept', () => {
-    it('should logout if a 401 error is caught and open popup', fakeAsync(() => {
```

Comment:
We should keep this testcase

Reply:
but it will fail since we removed the code that logs out when a 401 error is received in service, do we want to keep that condition?